ballerina/Ballerina.toml

```diff
@@ -0,0 +1,119 @@
+[package]
+org = "ballerina"
+name = "http"
+version = "@toml.version@"
+
+[[platform.java11.dependency]]
+path = "../native/build/libs/http-native-@project.version@.jar"
+
+
+[[platform.java11.dependency]]
+path = "./lib/mime-native-@stdlib.mimenative.version@.jar"
+groupId = "ballerina"
+artifactId = "mime-native"
+version = "@stdlib.mimenative.version@"
+
+
+[[platform.java11.dependency]]
+path = "./lib/netty-common-@netty.version@.jar"
+groupId = "io.netty"
+artifactId = "netty-common"
+version = "@netty.version@"
+
+
+[[platform.java11.dependency]]
+path = "./lib/netty-buffer-@netty.version@.jar"
+groupId = "io.netty"
+artifactId = "netty-buffer"
+version = "@netty.version@"
+
+
+[[platform.java11.dependency]]
+path = "./lib/netty-transport-@netty.version@.jar"
+groupId = "io.netty"
+artifactId = "netty-transport"
+version = "@netty.version@"
+
+
+[[platform.java11.dependency]]
+path = "./lib/netty-resolver-@netty.version@.jar"
+groupId = "io.netty"
+artifactId = "netty-resolver"
+version = "@netty.version@"
+
+
+[[platform.java11.dependency]]
+path = "./lib/netty-handler-@netty.version@.jar"
+groupId = "io.netty"
+artifactId = "netty-handler"
+version = "@netty.version@"
+
```

Comment:
Since we are directly pointing to the path of the jar, we can remove the other entries here. Check other places as well.
```suggestion
```